### ISSUE TITLE
Long waiting time between authentication and begin of operation (e.g. download)

### ISSUE THREAD

**Issue by Tjorbenn** [EGA-archive/ega-download-client#176]
**Title of the bug**
Long waiting time between authentication and begin of operation

*(e.g. Error with `--delete-temp-files`)*
e.g. download

**Description of the bug**
It takes a very long time (between 5-20 minutes) for the desired operation to start after the account has been authenticated.
This happens on multiple devices with different hardware and different software in different networks.

**Used versions** (please complete the following information)
 - Operating System version: macOS Monterey Version 12.6 / Windows 11 22H2
 - Python version: 3.10.6
 - PyEGA3 version: 4.0.5

**To Reproduce**

Steps to reproduce the behaviour that led you to the bug.

1. Trying to download the Dataset EGAD00001003335
```
pyega3 --connections 30  -cf /Users/tjorben/Documents/bachelor/credentials.json fetch EGAD00001003335 --output-dir /Users/tjorben/Documents/bachelor/validation
```
2. Authentication success appears immediately
3. Waiting for the start of the download

**Observed behaviour**

The start of the operation (download) takes 5-20 minutes

**Expected behaviour**

The start of the operation (download) starts in a reasonable time (immediately - 1 minute, the same speed the authentication takes).

**Screenshots and error messages**

[pyega3_output.log](https://github.com/EGA-archive/ega-download-client/files/9850617/pyega3_output.log)

**Comment by f6v** [comment]
I experience the same behavior. It was 7 minutes for me.

**Comment by JiayangZhou** [comment]
Same for me, waiting forever from Mac

**Comment by JiayangZhou** [comment]
`pyEGA3 - EGA python client version 4.0.5 (https://github.com/EGA-archive/ega-download-client)
[2022-11-16 22:28:02 +0100] Parts of this software are derived from pyEGA (https://github.com/blachlylab/pyega) by James Blachly
[2022-11-16 22:28:02 +0100] Python version : 3.7.10
[2022-11-16 22:28:02 +0100] OS version : Darwin Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:24 PDT 2021; root:xnu-8019.41.5~1/RELEASE_ARM64_T8101
[2022-11-16 22:28:02 +0100] MacOS version : 10.16
[2022-11-16 22:28:02 +0100] Server URL: https://ega.ebi.ac.uk:8052/elixir/data
[2022-11-16 22:28:02 +0100] Session-Id: 1420716916
[2022-11-16 22:28:02 +0100] Starting new HTTPS connection (1): ipinfo.io:443
[2022-11-16 22:28:03 +0100] https://ipinfo.io:443 "GET /json HTTP/1.1" 200 None
[2022-11-16 22:28:03 +0100] Starting new HTTPS connection (1): ega.ebi.ac.uk:8443
[2022-11-16 22:28:03 +0100] https://ega.ebi.ac.uk:8443 "POST /ega-openid-connect-server/token HTTP/1.1" 200 None
[2022-11-16 22:28:03 +0100] 
[2022-11-16 22:28:03 +0100] Authentication success for user 'xxx'
[2022-11-16 22:28:03 +0100] Starting new HTTPS connection (1): ega.ebi.ac.uk:8052
[2022-11-16 22:29:04 +0100] https://ega.ebi.ac.uk:8052 "GET /elixir/data/metadata/files/EGAF0000253238 HTTP/1.1" 500 None
[2022-11-16 22:29:04 +0100] Incremented Retry for (url='/elixir/data/metadata/files/EGAF0000253238'): Retry(total=19, connect=False, read=10, redirect=None, status=9)
[2022-11-16 22:29:04 +0100] Retry: /elixir/data/metadata/files/EGAF0000253238
[2022-11-16 22:29:04 +0100] Resetting dropped connection: ega.ebi.ac.uk`

**Comment by BigBella001** [comment]
You can hire Henryclarkethicalhacker for all your hacking needs which include clearing of criminal and driving records, credit hack fix, college grade changes, cloning phones, spying on anyone, hacking all social media accounts, etc,. Reach him via [Henryclarkethicalhacker @ gmail.com
 Whatsapp,,+12622367526.

**Comment by puva** [comment]
Same behavior. For 32 files from dataset EGAD00001008392, download took an average of 13 minutes to start (min 3, max 25 minutes).

**Comment by malloryfreeberg** [comment]
Thank you, all, for your patience while we worked on and released a new version of both [pyega3 (v5.0.0)](https://pypi.org/project/pyega3/) and the underlying API. Please update to the newest version of pyega3 (you can find the instructions [here](https://github.com/EGA-archive/ega-download-client#installation-and-update)) to enjoy the new features, including much faster authentication and authorisation steps.

This GH issue will be closed, but if you continue to experience the same problem, feel free to re-open it. If you experience any other problems, please email the EGA Helpdesk at helpdesk@ega-archive.org and attach the output log produced by pyega3 (pyega3_output.log). The Helpdesk team can then provide further assistance. Thank you!
